.github/workflows/python-test.yml

```diff
@@ -43,4 +38,4 @@ jobs:
       - name: Test with pytest
         run: |
           pip install pytest
-          pytest tests/
\ No newline at end of file
+          python -m pytest --typeguard-packages=zigzag tests/
```

Comment:
please add a newline to every file you save :) 
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files